app/translations/dutch.js

```diff
@@ -0,0 +1,396 @@
+  privateKeyLabel: 'PRIVATE KEY',
+  encryptedKeyLabel: 'VERSLEUTELDE KEY',
+  addressLabel: 'PUBLIEK ADRES',
+  splitKeyLabel: 'GESPLITSTE SLEUTEL',
+  recoverWalletLabel: 'HERSTEL WALLET',
+  print: 'Print',
+  generateQrCodes: 'Genereer QR-Codes',
+  copyCodeImage: 'Kopieer Code Image',
+}
+
+const DASHBOARD = {
+  dashboardBalancePanelLabel: 'Token overzicht',
+  dashboardAssetsPanelLabel: 'Systeem assets',
+  dashboardAssetsTotal: 'TOTAAL',
+  dashboardMarketDataLabel: 'Marktdata',
+  dashboardValueLabel: 'Totale walletwaarde',
+  dashboardAddressLabel: 'Adres:',
+  dashboardPriceNotAvailable: 'N.v.t.',
+  dashboardGasClaimButton: 'Claim {amount} GAS',
+  dashboardManageWallets: 'Beheer wallets',
+  dashboardRefresh: 'Verversen',
+  dashboardTokenBalancesToken: 'Token',
+  dashboardMarketData1Day: '1 DAG',
+  dashboardMarketData1Week: '1 WEEK',
+  dashboardMarketData1Month: '1 MAAND',
+}
+
+const SIDEBAR = {
+  sidebarWallet: 'Wallet',
+  sidebarActivity: 'Activiteit',
+  sidebarSend: 'Verzenden',
+  sidebarReceive: 'Ontvangen',
+  sidebarContacts: 'Contacten',
+  sidebarTokenSale: 'Token Verkoop',
+  sidebarNews: 'Nieuws',
+  sidebarSettings: 'Instellingen',
+  sidebarLogout: 'Uitloggen',
+  sidebarCurrentBlock: 'HUIDIG BLOK',
+}
+
+const MANAGE_WALLETS = {
+  manageWalletsLabel: 'Beheer wallets',
+  manageWalletsImport: 'Importeren',
+  manageWalletsCreate: 'Maak',
+  manageWalletsEdit: 'Aanpassen',
+  manageWalletsEditWallet: 'Pas wallet aan',
+  manageWalletsEditWalletInstructions: 'Wijzig details',
+  manageWalletsEditWalletNameLabel: 'WALLET NAAM',
+  manageWalletsEditWalletNamePlaceholder: 'Wallet naam',
+  manageWalletsEditWalletAddressLabel: 'WALLET ADRES',
+  manageWalletsEditWalletSave: 'Wijzigingen opslaan',
+}
+
+const ACTIVITY = {
+  activityAddAddress: 'Toevoegen',
+  activityViewTx: 'Bekijken',
+  activityPageLabel: 'Alle activiteiten',
+  activityExport: 'Exporteren',
+}
+
+const RECEIVE = {
+  recieveSelectMethod: 'Selecteer stortingsmethode',
+  receiveAssetsAddressLabel: 'Jouw publieke adres',
+  receivePageLabel: 'Ontvang assets',
+  receiveYourAddressTabLabel: 'JOUW ADRES',
+  receiveCopyCodeButton: 'Copy Code Afbeelding',
+  receiveDisclaimer:
+    'Stuur alleen assets die <b>compatibel zijn met de NEO-blockchain (NEO, GAS, etc.)</b>. Het verzenden van andere assets zal resulteren in permanent verlies.',
+  receiveRequestTabAssets: 'VERZOEK ASSETS',
+  recieveWhyUseQRLabel: 'Waarom een QR-Code gebruiken??',
+  receiveQRExplanation:
+    '<p>Ooit assets naar het verkeerde adres gestuurd vanwege een foutief karakter in het wallet-adres?</p><p>Zo niet, dan heb je mazzel - maar het gebeurt veel te vaak.</p><p>Hier bij CoZ willen we ervoor zorgen dat mensen die jou betalen, het juiste adres krijgen. J kunt een QR-code genereren om assets aan te vragen, zodat ze jou kunnen helpen. </p> <p> Elke code die je genereert, bevat uw openbare wallet-adres, een aantal en een referentie - allemaal door jou ingesteld. </p>',
```

Comment:
```suggestion
    '<p>Ooit assets naar het verkeerde adres gestuurd vanwege een foutief karakter in het wallet-adres?</p><p>Zo niet, dan heb je mazzel - maar het gebeurt veel te vaak.</p><p>Hier bij CoZ willen we ervoor zorgen dat mensen die jou betalen, het juiste adres krijgen. Je kunt een QR-code genereren om assets aan te vragen, zodat ze jou kunnen helpen. </p> <p> Elke code die je genereert, bevat je openbare wallet-adres, een aantal en een referentie - allemaal door jou ingesteld. </p>',
```
Typo + mix of informal and formal. Given that you used informal pretty much everywhere else, I suggest to keep it here as well.